__tests__/mainTestSuite.test.js

```diff
@@ -169,3 +169,70 @@ describe('GET /api/articles/:article_id/comments', () => {
     })
   });
 });
+
+describe('POST /api/articles/:article_id/comments', () => {
+  it('should add a new comment to an article referenced by its id', () => {
+    return request(app)
+    .post('/api/articles/2/comments')
+    .send({
+      username: 'butter_bridge',
+      body: 'Commenting, commenting everyone loves to comment'
+    })
+    .expect(201)
+    .then((result) => {
+      const body = result.body
+      expect(body.addedComment.length).toBe(1)
+      body.addedComment.forEach((comment) => {
+        expect(typeof comment['author']).toBe('string')
+        expect(typeof comment['body']).toBe('string')
+      })
+    })
+  });
+  it('should respond with an appropriate error message when given a valid but non-existent id', () => {
+    return request(app)
+    .post('/api/articles/99999/comments')
+    .send({
+      username: 'butter_bridge',
+      body: 'Commenting, commenting everyone loves to comment'
+    })
+    .expect(404)
+    .then((result) => {
+      expect(result.body.msg).toBe('Invalid inputted data')
```

Comment:
I'd consider this error message, how useful is it? The input is totally valid, so is the id. It just doesn't exist.